frontend/src/utils/auth.js

```diff
@@ -9,12 +9,17 @@ export const getToken = () => {
   return window.localStorage.getItem(localStorageKey);
 };
 
-// AUTHENTICATION
-export const isAuthenticated = () => {
-  const token = getToken();
-  if (token) {
-    return true;
+export const checkForToken = () => {
+  if (getToken()) {
+    const optionsToken = {
+      headers: {
+        Authorization: `Bearer ${getToken()}`,
+        "Content-Type": "application/json"
+      }
+    };
+
+    return optionsToken;
   }
 
-  return false;
+  return null;
 };
```

Comment:
try to keep the code as readable as possible

```
export const checkForToken = () => {
  const optionsToken = {
    headers: {
      Authorization: `Bearer ${getToken()}`,
      "Content-Type": "application/json"
    }
  };

  return getToken()
    ? optionsToken;
    : null;
};
```

Reply:
done